website/templates/project/statistics.mako

```diff
@@ -5,6 +5,16 @@
   <h2 class="text-300">Analytics</h2>
 </div>
 
+<div id="hiddenFlag" class="ad_block_display">
+  <div class="m-b-md p-md osf-box-lt box-round text-center">The use of adblocking software may prevent site analytics from loading properly.</div>
+</div>
+<script src="/static/js/ads.js"></script>
+<script>
+  if( window.canRunAds === undefined ){
+    document.getElementById('hiddenFlag').style.display = "block";
```

Comment:
When you click the link, look for the gold highlighted line. It points to line 6 in ads.js.